dev/cognitiveservices/data-plane/Language/generate-questionanswers.json

```diff
@@ -0,0 +1,643 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Cognitive Language Service - Generate Question and Answer pairs ",
```

Comment:
Are these effectively new endpoints for Question Answering? What about calling the file something like questionanswering-generate.json to better pair with the other, related files with a similar convention?